src/components/Header.scss

```diff
@@ -24,7 +24,9 @@ $header-icon-size: ($header-height * 0.5);
 
     a,
     h1 {
-        font-size: $font-size-regular;
+        font-size: $font-size-header;
+        font-family: $display-font-stack;
+        font-weight: lighter;
```

Comment:
It might make more sense to set an explicit weight, instead of relying on the weight of the parent element.